srcs/cd.c

```diff
@@ -6,12 +6,27 @@
 /*   By: tgrivel <marvin@42lausanne.ch>             +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/06/12 17:01:54 by tgrivel           #+#    #+#             */
-/*   Updated: 2022/06/27 17:41:59 by lgyger           ###   ########.fr       */
+/*   Updated: 2022/06/28 13:10:31 by lgyger           ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
+static void	why(char *path)
+{
+	struct stat	file;
+
+	if (stat(path, &file) == -1)
+	{
+		printf("cd: %s: no such file or directory\n", path);
+		return ;
+	}
+	if (!(file.st_mode & S_IFDIR))
```

Comment:
What is that ?
```c
& S_IFDIR
```

Reply:
And will return one if the flag is present in the binary number on the left and S_IFDIR will OR ed in the flag if it is a directionary